src/NuGet.Core/NuGet.Versioning/FloatRange.cs

```diff
@@ -162,25 +163,25 @@ public bool Satisfies(NuGetVersion version)
         /// <summary>
         /// Parse a floating version into a FloatRange
         /// </summary>
-        public static FloatRange Parse(string versionString)
+        public static FloatRange? Parse(string versionString)
         {
-            _ = TryParse(versionString, out FloatRange range);
+            _ = TryParse(versionString, out FloatRange? range);
 
             return range;
         }
```

Comment:
This is definitely a bug, lol :D 
I think we should throw rather than annotate, but I'm open to argument against that. 

I wouldn't expected too many people to be using this API directly.